### ISSUE TITLE
Tillerino not able to connect to api for !u command

### ISSUE THREAD

**Issue by richp2k** [Tillerino/Tillerinobot#185]
I just noticed this now, but I think this has been happening since the osu!catch SR and pp rework due to osu! api changes.
![screenshot098](https://user-images.githubusercontent.com/66353518/83605903-bcad7700-a546-11ea-8ca8-92e57fa3b305.png)

**Comment by omkelderman** [comment]
that error message is actually a little misleading. When I made that I just reused the error message from other parts of the bot that dealt with osu! API communication.

The !u command however, communicates with the osutrack api, and the osutrack website (https://ameobea.me/osutrack/) has an invalid HTTPS certificate right now, hence why !u is failing atm. It will continue to give that error until the osutrack website gets a valid certificate again.

in short: it has nothing to do with what the error message suggest, it is simply unable to connect to osutrack due to a invalid cert

**Comment by MrMorkat** [comment]
We might want to change that message then... If i can find the time i will do it tomorrow

**Comment by omkelderman** [comment]
That's not as easy as you might think, a connection exception is thrown (either in osu api code, or osutrack api code) and when its catched it assumes osu api (since at first that was the only thing it was used for)

Basically at the place where the message text is chosen you can't see if it's an osu api error or osutrack api error)

**Comment by Tillerino** [comment]
You would throw a UserException where the osu!track API is called, but also
need a special message (and translations).

Has the certificate been updated? The website looks fine.

**Comment by MrMorkat** [comment]
ye ameo fixed it an hour ago or something

**Comment by Ameobea** [comment]
Yeah sorry about that - this was a while ago and I've set up an auto-update script for my certs going forward.

**Comment by Tillerino** [comment]
No harm done. They're all free services :)

And we should have closed this a while ago, I guess
